hedera-node/hedera-mono-service/build.gradle.kts

```diff
@@ -76,6 +76,7 @@ dependencies {
 
     testImplementation(testLibs.bundles.testing)
     testImplementation(testLibs.classgraph)
+    testImplementation(testLibs.google.truth)
```

Comment:
We should use AssertJ over Truth if possible. Our agreed upon standard is AssertJ.

Reply:
I'm with you. Especially that Truth is not working well with the Java module system. But I just moved tests that needs it ;) Already created an issue for that some days ago: https://github.com/hashgraph/hedera-services/issues/4429